sdk/elixir/runtime/dagger.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "elixir-sdk",
```

Comment:
I see typescript use capital case but in Python use kebab case. I'm not sure which one we should use but I follow Python SDK. 😂

Reply:
Good point, I think kebab is the way to go, will see that it's normalized 🙂

Reply:
Yeah. But I need to test more on this case because currently, I just normalize this name to lower case and replace - to _ to use in Elixir package name. :(

Reply:
Fixed with `strcase`. :)